### DISCUSSION TITLE
Tagging bus stops, what does the number mean? Like a serial number for the specific stop, or the number of the bus lines using it?

### DISCUSSION THREAD

**Discussion by leethax666** [streetcomplete/StreetComplete#4848] [Q&A] [answered]
Title mostly. I can easily add the number for the bus lines using it, but none of them have had specific individual numbers. The closest is the local bus system has a feature you can text with a number code for the stop, and it'll reply with the next arrival

**Comment by BalooUriza** [comment, marked as answer]
That is the bus stop number or ref=* for the stop.

**Reply by leethax666** [reply]
thank you! i should just turn it off then

**Reply by riQQ** [reply]
In which country is this happening? The quest possibly should be disabled there, if no bus stops have refs in your country.

**Reply by matkoniecz** [reply]
Note that country may typically have `ref` numbered bus stops and some city/town/region without them, in such case answering "no ref here" is still useful.

**Reply by leethax666** [reply]
some of them have them so i know it's a thing, but most stops i've been at, it's either not clear or worn off, would "no ref here" apply to that?

**Reply by matkoniecz** [reply]
See https://www.openstreetmap.org/node/9212238798 - it will mark `ref:signed=no` so if reference is assigned and labelling is missing, then StreetComplete answer is still correct.

You can check tagging also by disabling internet on phone, solving quest and undoing edit and looking at applied tags, then confirming undo and enabling internet. Disabling internet part blocks applying test data to OSM even as temporary edit.

You can also look at https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_ref/AddBusStopRef.kt#L40

**Reply by riQQ** [reply]
An alternative to turning off the internet temporarily is to temporarily disable auto upload in the StreetComplete settings.

**Comment by BalooUriza** [comment]
On Sun, Feb 26, 2023 at 10:41 PM belial ***@***.***> wrote:

> Title mostly. I can easily add the number for the bus lines using it, but
> none of them have had specific individual numbers. The closest is the local
> bus system has a feature you can text with a number code for the stop, and
> it'll reply with the next arrival
>
That is the ref=* of the bus stop, right there.  That's the bus stop number
you enter when SC asks for one.

> TMessage ID: ***@***.***
> com>
>
